packages/fluxible-router/.npmignore

```diff
@@ -1,5 +1,8 @@
 .npmignore
 /artifacts/
 /docs/
+/src/
```

Comment:
We run `dist` before `pack` and the dist files are present in the generated tarball.

```bash
$ npm pack

> fluxible-router@1.5.1 prepublish fluxible/packages/fluxible-router
> npm run dist


> fluxible-router@1.5.1 dist fluxible/packages/fluxible-router
> ../../node_modules/.bin/babel src -d dist

src/addons/HistoryWithHash.js -> dist/addons/HistoryWithHash.js
src/addons/index.js -> dist/addons/index.js
src/lib/History.js -> dist/lib/History.js
src/lib/NavLink.js -> dist/lib/NavLink.js
src/lib/RouteStore.js -> dist/lib/RouteStore.js
src/lib/createNavLinkComponent.js -> dist/lib/createNavLinkComponent.js
src/lib/handleHistory.js -> dist/lib/handleHistory.js
src/lib/handleRoute.js -> dist/lib/handleRoute.js
src/lib/navigateAction.js -> dist/lib/navigateAction.js
fluxible-router-1.5.1.tgz

$ tar -tf fluxible-router-1.5.1.tgz 
package/package.json
package/README.md
package/index.js
package/dist/addons/HistoryWithHash.js
package/dist/addons/index.js
package/dist/lib/History.js
package/dist/lib/NavLink.js
package/dist/lib/RouteStore.js
package/dist/lib/createNavLinkComponent.js
package/dist/lib/handleHistory.js
package/dist/lib/handleRoute.js
package/dist/lib/navigateAction.js
package/CHANGELOG.md
package/LICENSE.md
```